feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create-dialog/customer-ticketing-create-dialog.component.ts

```diff
@@ -1,29 +1,121 @@
-import { Component, OnInit } from '@angular/core';
-import { AssociatedObject, Category } from '@spartacus/customer-ticketing/root';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import {
+  AssociatedObject,
+  Category,
+  TicketStarter,
+} from '@spartacus/customer-ticketing/root';
 import { FormUtils } from '@spartacus/storefront';
-import { Observable } from 'rxjs';
+import { Observable, Subscription } from 'rxjs';
 import { CustomerTicketingDialogComponent } from '../../shared/customer-ticketing-dialog/customer-ticketing-dialog.component';
 @Component({
   selector: 'cx-customer-ticketing-create-dialog',
   templateUrl: './customer-ticketing-create-dialog.component.html',
 })
 export class CustomerTicketingCreateDialogComponent
   extends CustomerTicketingDialogComponent
-  implements OnInit
+  implements OnInit, OnDestroy
 {
   ticketCategories$: Observable<Category[]> =
     this.customerTicketingFacade.getTicketCategories();
   ticketAssociatedObjects$: Observable<AssociatedObject[]> =
     this.customerTicketingFacade.getTicketAssociatedObjects();
+  subscription: Subscription;
+  ticketStarter: TicketStarter;
+  ticketCategories: Array<Category>;
+  ticketAssociatedObjects: Array<AssociatedObject>;
```

Comment:
```suggestion

```

Do we need `ticketCategories` and `ticketAssociatedObjects`?

Reply:
Removed.